harness/determined/common/api/authentication.py

```diff
@@ -81,7 +82,7 @@ def _init_session(
             token = util.get_container_user_token()
 
         if token is not None:
-            return Session(session_user, token)
+            return Session(session_user, token, self.token_store.get_active_user_id())
```

Comment:
Question: what happens to the currently logged in CLI users after upgrading to this?

Reply:
this question still stands... do we need to shim the old token store to fetch the user id for existing logins?..

Reply:
what do you think about using the `users/me` call -- i think this should only occur one time, in the case where there's a token but no `active_user_id` in the token store?

Reply:
sounds like a plan to me